.circleci/config.yml

```diff
@@ -126,10 +126,11 @@ jobs:
                 echo "Skipping acceptance tests for forked PRs; marking step successful."
                 circleci step halt
             fi
-            primary_context=$(terraform output -state ../../terraform/gke/terraform.tfstate -json | jq -r .context_names.value[0])
-            secondary_context=$(terraform output -state ../../terraform/gke/terraform.tfstate -json | jq -r .context_names.value[1])
+            eval "$(echo export primary_kubeconfig=$(terraform output -state ../../terraform/gke/terraform.tfstate -json | jq -r .kubeconfigs.value[0]))"
```

Comment:
why do you need eval here? my bash foo is lacking

Reply:
haha, it's because the output of terraform contains another env variable (`$HOME`). If I don't eval it, it'll just be passed as a string because it's stored inside another variable.